src/transformers/models/roberta/modeling_flax_roberta.py

```diff
@@ -977,8 +977,8 @@ def __call__(
 
         logits = self.qa_outputs(hidden_states)
         start_logits, end_logits = logits.split(self.config.num_labels, axis=-1)
-        start_logits = start_logits.squeeze(-1)
-        end_logits = end_logits.squeeze(-1)
+        start_logits = start_logits.squeeze(-1).contiguous()
+        end_logits = end_logits.squeeze(-1).contiguous()
```

Comment:
I don't think this should be here, this is a Flax model

Reply:
I'm really sorry that I don't know much about Flax model, but I have made some modifications. Many thanks!